src/ophyd_async/fastcs/panda/_hdf_panda.py

```diff
@@ -19,7 +19,9 @@ def __init__(
         name: str = "",
     ):
         # This has to be first so we make self.pcap
-        connector = fastcs_connector(self, prefix)
+        connector = fastcs_connector(
+            self, prefix, error_hint="Is PandABlocks-ioc at least version 0.11.4?"
```

Comment:
This is very likely to be forgotten about and not updated. Some suggestions:
* Should: Pull it out into a `MINIMUM_PANDA_IOC` variable - this should at least make it stand out more
* Do you have any where in you're PR/release process you could add a reminder to update it if necessary?
* We could just change the text to say "the latest panda IOC version" - I don't really like this but if it's forgotten about and suggests an old version it's actually worse